lib/iris/tests/unit/analysis/test_PERCENTILE.py

```diff
@@ -97,6 +98,170 @@ def test_masked_2d_multi(self):
         self.assertArrayAlmostEqual(actual, expected)
 
 
+class LazyMixin:
```

Comment:
Before I read this file, I was wondering if we could leverage the existing tests and just run them once with real data and once with lazy data to show that the effect is identical - is that possible? We could also cover off real fast percentile aggregation, which I think we support but isn't currently tested?

Reply:
This happened, it was pretty sweet